extensions/wikia/FeedsAndPosts/JustUpdatedAPIProxy.class.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Wikia\FeedsAndPosts;
+
+class JustUpdatedAPIProxy {
+
+	const LIMIT = 4;
+
+	private function requestAPI($params) {
```

Comment:
Our [PHP guidelines](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md) are a superset of the MediaWiki guidelines, and those call for spaces inside of all of those parentheses: https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP#Spaces. So we should probably update those.